src/graphics/text.rs

```diff
@@ -353,13 +353,11 @@ impl Text {
     pub fn dimensions(&self, context: &Context) -> (u32, u32) {
         if let Ok(metrics) = self.cached_metrics.read() {
             match (metrics.width, metrics.height) {
```

Comment:
Nitpick; this could be an `if let`:
```rust
if let (Some(width), Some(height)) = (metrics.width, metrics.height) {
    return (width, height);
}
```

Reply:
@Ratysz fixed